packages/webpack/src/runtime/runtime-namespace.ts

```diff
@@ -7,6 +7,7 @@ type DebugTools = {
 
 type RequiredProperty<T, K extends keyof T> = Omit<T, K> & Required<Pick<T, K>>
 export interface RuntimeNamespace {
+  scuttling: {scuttle: <T, U>(arg1: T, arg2: U) => {}}
```

Comment:
1. Why do we have generics here?
2. Do you mean to return type `{}`, which means "an object which is not required to have properties but could."

Reply:
1. is this better? 5eaca0062
2. i want it to return nothing, what should i use then?